options/locale/locale_en-US.ini

```diff
@@ -1710,7 +1710,7 @@ pulls.delete.text = Do you really want to delete this pull request? (This will p
 
 milestones.new = New Milestone
 milestones.closed = Closed %s
-milestones.update_ago = Updated %s ago
+milestones.updated = Updated
```

Comment:
We shouldn't do this.

It breaks the i18n translation, because different languages have different date/time syntax.

Reply:
Right, so just `Updated %s` for English is better just like `closed`. Because other languages may be `%s xxx`.

Reply:
To avoid making the i18n system more complex, a quick path could be:

`milestones.updated = "Updated: "`

The colon helps to separate the sentence, it works in all languages.